aaf_walking_group/action/WalkingGroup.action

```diff
@@ -0,0 +1,5 @@
+string group
+string[] parameters
```

Comment:
These parameters/values need to be there? I'm asking because they all would need to be created by the routine, won't they? Is this needed?